### PR TITLE
Add source view for store credits

### LINKED ISSUE #193: Missing partial for store credits in admin.
Should add something at `spree/admin/payments/source_views/_storecredit` otherwise store credit payments will result in server errors when viewing them in the admin.

### PR DESCRIPTION
**Pull request by adammathys**
Pretty simple, just shows some basic info. Mostly to stop the exception from happening.

Closes #193

### CODE REVIEW

**Comment by jhawthorn** [comment]
This needs a test

File: backend/spec/features/admin/payments/store_credits_spec.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  let(:order) { FactoryGirl.create(:completed_order_with_totals) }
+  let(:payment) do
+    FactoryGirl.create(
+      :store_credit_payment,
+      order: order,
+      amount: 20
+    )
+  end
+
+  it "vieweing a store credit payment" do
```
Comment (magnusvk) [inline backend/spec/features/admin/payments/store_credits_spec.rb]: vieweing -> viewing. Magnus, the German master of nitpicking!

**Comment by magnusvk** [comment]
:+1:

**Comment by jhawthorn** [comment]
:+1:

**Pull request merged**